listenbrainz_spark/stats/tests/test_init.py

```diff
@@ -0,0 +1,37 @@
+	#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Dec 15 20:12:56 2019
+
+@author: sarthak
+"""
+
+import datetime
+from listenbrainz_spark import utils
+from pyspark.sql import Row
+from listenbrainz_spark import stats
+from listenbrainz_spark.tests import SparkTestCase
+
+
+class InitTestCase(SparkTestCase):
+
+    def test_replace_days(self):
+        self.assertEqual(stats.replace_days(datetime.datetime(2019,5,12),13),datetime.datetime(2019,5,13,0,0))
+        
+    def test_adjust_months(self):
+        self.assertEqual(stats.adjust_months(datetime.datetime(2019,5,12),3,True),datetime.datetime(2019,2,12,0,0))
```

Comment:
You don't need to pass *true* to *adjust_months* since the default value is *true*. If you want to assign a value other than the default value to shift_backwards then pass it i.e false. Also, there are two ways by which date can be manipulated in adjust_months, you should check for both. Something like this: 

`self.assertEqual(stats.adjust_months(datetime.datetime(2019,5,12), 3), datetime.datetime(2019,2,12))`
`self.assertEqual(stats.adjust_months(datetime.datetime(2019,5,12), 3, False), datetime.datetime(2019,8,12))`

Also, I find it more readable to break this code into pieces. Like here:
`d1 = stats.adjust_months(datetime.datetime(2019,5,12), 3, False)`
`d2 =  datetime.datetime(2019,8,12)`
`self.assertEqual(d1, d2)`